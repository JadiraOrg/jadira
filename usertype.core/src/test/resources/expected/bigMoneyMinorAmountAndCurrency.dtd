<!ELEMENT dataset (
    BIGMONEYMINORAMOUNTANDCURRENCY*)>

<!ELEMENT BIGMONEYMINORAMOUNTANDCURRENCY EMPTY>
<!ATTLIST BIGMONEYMINORAMOUNTANDCURRENCY
    ID CDATA #REQUIRED
    MY_CURRENCY CDATA #IMPLIED
    MY_AMOUNT CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

