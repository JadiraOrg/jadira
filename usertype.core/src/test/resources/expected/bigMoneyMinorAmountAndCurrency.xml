<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyMinorAmountAndCurrency.dtd">
<dataset>
  <bigMoneyMinorAmountAndCurrency ID="0" MY_CURRENCY="USD" MY_AMOUNT="10000" NAME="test_0"/>
  <bigMoneyMinorAmountAndCurrency ID="1" MY_CURRENCY="USD" MY_AMOUNT="10010" NAME="test_1"/>
  <bigMoneyMinorAmountAndCurrency ID="2" MY_CURRENCY="EUR" MY_AMOUNT="99" NAME="test_2"/>
  <bigMoneyMinorAmountAndCurrency ID="3" MY_CURRENCY="EUR" MY_AMOUNT="-99" NAME="test_3"/>
  <bigMoneyMinorAmountAndCurrency ID="4" NAME="test_4"/>
</dataset>